src/cloud/project/routes.md

```diff
@@ -141,6 +141,8 @@ In the following examples, the route configuration routes the apex domain and th
     to: "http://{default}/"
 ```
 
+In this example, the server responds directly to a request of the form `http://example.com/hello`, issuing a _301 redirect_ for requests with the URL pattern `http://www.example.com/mypath`. These requests redirect to the apex domain, for example `http://example.com/mypath`.
```

Comment:
To avoid a strange appearance by ending a code formatted snippet with a period, it's cleaner to use a colon. Otherwise, looks like a period floating out there at the end of the URL.

```suggestion
In this example, the server responds directly to a request of the form `http://example.com/hello`, issuing a _301 redirect_ for requests with the URL pattern `http://www.example.com/mypath`. These requests redirect to the apex domain, for example: `http://example.com/mypath`
```